server/filestore.go

```diff
@@ -410,6 +411,12 @@ func (fs *fileStore) recoverMsgBlock(fi os.FileInfo, index uint64) *msgBlock {
 	}
 	defer file.Close()
 
+	if fi, _ := file.Stat(); fi != nil {
+		mb.rbytes = uint64(fi.Size())
+	} else {
+		return nil
```

Comment:
So the `file` representing a message block has been opened just above, but here we say if `file.Stat()` fails basically, then we ignore? It may actually be better to continue no?

Reply:
Stat should always succeed, just being overly cautious but if it does fail we should not continue imo.